jupyter-notebooks/udm2/udm2.ipynb

```diff
@@ -6,18 +6,22 @@
    "source": [
     "# Usable Data Mask (UDM2) Visualization and Masking\n",
     "\n",
-    "The Usable Data Mask (UDM2) is an iteration upon the Unusable Data Mask (UDM) product. It is available for many products and it contains the UDM as one of its bands. Specifications for the UDM2 are given in the [Imagery Spec Sheet](https://assets.planet.com/docs/combined-imagery-product-spec-april-2019.pdf) and the [UDM-2 API documentation](https://developers.planet.com/docs/api/udm-2/).\n",
+    "The Usable Data Mask (UDM2) is an iteration upon the Unusable Data Mask (UDM) product. It is available for many products and it contains the UDM as one of its bands. Specifications for the UDM2 are given in the [Imagery Spec Sheet](https://assets.planet.com/docs/Planet_Combined_Imagery_Product_Specs_letter_screen.pdf) and the [UDM2 API documentation](https://developers.planet.com/docs/api/udm-2/).\n",
     "\n",
-    "The UDM2 is a multi-band image. The first six bands are boolean bands that encode the absence or presence of different image usability concerns such as haze, clouds, and cloud shadows. The sixth band provides the confidence in the usability assessments as integer values between 0 and 100. The seventh band is the Unusable Data Mask, which is described in the Imagery Spec Sheet referenced above and explored in the [UDM notebook](../udm/udm.ipynb).\n",
+    "The UDM2 is a multi-band image. The first six bands are boolean bands that encode the absence or presence of different image usability concerns such as haze, clouds, and cloud shadows. The seventh band provides the confidence in the usability assessments as integer values between 0 and 100. The eighth band is the Unusable Data Mask, which is described in the Imagery Spec Sheet referenced above and explored in the [UDM notebook](./udm.ipynb).\n",
     "\n",
-    "In this notebook, we demonstrate how to visualize the UDM2 bands and how to convert the UDM2 into two masks: masking all unclear pixels and masking cloud and shadow pixels."
+    "In this notebook, we demonstrate how to visualize the UDM2 bands and how to convert the UDM2 into two masks: masking all unclear pixels and masking cloud and shadow pixels.\n",
+    "\n",
+    "### Note on UDM2.1\n",
+    "\n",
+    "After November 2023, with the launch of UDM2.1, the heavy haze class has been deprecated. A new computer vision model trained at Planet returns more accurate class masks, and includes only a single haze class. For backwards compatibility, `light_haze` will store this haze detection, whereas `heavy_haze` will return zero for all pixels going forward."
```

Comment:
maybe we can link to the product spec here (I know it's not available yet).

I also think you can be a bit clearer about the fact that scenes captured before November 29, 2023 are UDM2.0 and have a valid heavy haze band, whereas after the release date, the UDM2.1 masks will all have a zeroed heavy_haze band. 

Maybe something like:

> "An updated Usable Data Mask, UDM2.1, was released on November 29, 2023: all imagery collected after this date will be processed with UDM2.1. For the new product version, the computer vision model that powers UDM2 has been updated to deliver more accurate cloud masks. The UDM2.1 includes only a single haze class, and the heavy haze class has been deprecated.  For backwards compatibility, the `light_haze` band will store the new haze information. Pixels will never be classified as Heavy Haze with UDM2.1, and the `heavy_haze` band will contains zeros for all pixels. All imagery collected before the release date has been processed with UDM2.0 cloud masks, and contains both haze bands."